lib/main.dart

```diff
@@ -175,19 +176,28 @@ class _MyAppState extends State<MyApp> {
 
   /// Initializing the mainScreen window to 1 to show the events by default after app in opened.
   late int mainScreenQuickActionindex = 0;
+  Timer? timer;
   @override
   void initState() {
+    print("CLDD: let's called -------------------------------------");
+
+    // Timer.periodic(const Duration(seconds: 300), (timer) {
+    //   FetchPluginList();
+    // });
+    timer = Timer.periodic(
```

Comment:
This will entirely break the app.

1. There is no need to create a system-wide timer. It can be local to the MainScreen.
2. You are fetching plugins unconditionally. This will cause a null exception when the user has not entered org URL yet. There was a bugfix regarding this sometime ago. This code change will reintroduce that bug.

Reply:
Users can't enter without having the org URL.

Reply:
Nothing is getting "breaked" on my side.

Reply:
> Nothing is getting "breaked" on my side.

That is because you have not performed an E2E testing after clearing the data.